app/content/drill/[id]/leaderboard.js

```diff
@@ -13,16 +14,20 @@ export default function Leaderboard({ leaderboardData, drillId }) {
     return drillsData.teams["1"].users[userId];
   };
 
+  orderedLeaderboard.map((user) => (
+    console.log("user.score:", bestScores)
```

Comment:
btw, Frank and I figured out that the calculations of the `orderedLeaderboard` are always wrong, calculating infinity instead of an actual value @hannacol